src/Core/Grid/Query/AuthorizedApplicationQueryBuilder.php

```diff
@@ -108,10 +108,9 @@ private function applyAssociatedQueries(QueryBuilder $queryBuilder): void
     private function appendActiveApiAccessQuery(QueryBuilder $queryBuilder): void
     {
         $shopQueryBuilder = $this->getQueryBuilder()
-            ->select('count(api.active)')
+            ->select('count(api.enabled)')
             ->from($this->dbPrefix . 'api_access', 'api')
-            ->where('api.id_authorized_application = aa.id_authorized_application')
-            ->andWhere('api.active = 1');
+            ->Where('api.enabled = 1');
```

Comment:
You removed this condition because the relationship has been removed in the ApiAccess entity right? This changes the behavior of the grid, but it's supposed to be completely removed soon anyway so as long as tests do not shout I guess we don't really care